src/__snapshots__/footer.test.jsx.snap

```diff
@@ -207,7 +207,7 @@ exports[`Footer exists 1`] = `
           items={
             Array [
               <span>
-                © Meetup 2018
+                © Meetup 2019
```

Comment:
lol this snapshot was failing.. thought I'd just update it

Reply:
I thiiink we should just change the snapshot.. see here https://meetuphq.slack.com/archives/C099CDFJS/p1546441880004900

Reply:
^ this is jackie hi

Reply:
https://github.com/meetup/meetup-web-components/blob/master/src/Footer.jsx#L184 <- this is why it was failing. I think leave it for now and we'll sort it out